components/azure/azureml/aml-register-model/component.yaml

```diff
@@ -0,0 +1,30 @@
+name: Azure ML Register Model 
+description: A Kubeflow pipeline component register model with Azure ML.
+inputs:
+- {name: base_path, type: String}
+- {name: model_file, type: String}
+- {name: model_name, type: String}
+- {name: tenant_id, type: String}
+- {name: service_principal_id, type: String}
+- {name: service_principal_password, type: String}
+- {name: subscription_id, type: String}
+- {name: resource_group, type: String}
+- {name: workspace, type: String}
+- {name: run_id, type: String}
+
```

Comment:
Good components have outputs. This allows chaining components together to make a pipeline.
When creating a model, you can return the model name/path/URL.
When pushing the model for serving you can return the serving service endpoint that can be used to get model predictions.

Reply:
Good point added output for model name.